### PR TITLE
PRO-109 Visual fixes march

### PR DESCRIPTION
**Pull request by divyaBacancy**
Completed small UI changes in modals

https://user-images.githubusercontent.com/66996635/167590006-dfbdca4f-7a63-4179-846e-6a867e2c9659.mp4

### CODE REVIEW

File: src/components/legacy/Button/Button.js [hunk 1]
```diff
@@ -178,41 +187,42 @@ const getLabelTopMargin = (props) => {
...
 
 const ButtonText = styled(BaseText)`
   color: ${contentColor};
-  font-size: ${props => getButtonFontSize(props)}px;
-  line-height: ${props => getButtonFontSize(props)}px;
-  margin-top: ${props => getLabelTopMargin(props)}px;
+  font-size: ${(props) => getButtonFontSize(props)}px;
+  line-height: ${(props) => getButtonFontSize(props)}px;
+  margin-top: ${(props) => getLabelTopMargin(props)}px;
+  // font-family: ${appFont.medium};
```
Comment (IAmKio) [inline src/components/legacy/Button/Button.js:225]: Is this line dead?

File: src/locales/zh-CN/common.json [hunk 2]
```diff
@@ -122,6 +122,7 @@
     "added": "添加",
     "adding": "添加",
     "all": "全部",
+    "allNetwork":"所有网络",
```
Comment (IAmKio) [inline src/locales/zh-CN/common.json:125]: Are you using Google Translae for his?

File: src/screens/AddCash/AddCash.js [hunk 3]
```diff
@@ -65,11 +65,16 @@ import AddCashValueInputAccessoryHolder, {
...
 
+let visibleModal = false;
 const AddCash = () => {
   const navigation = useNavigation();
   const dispatch = useDispatch();
   const [value, setValue] = React.useState('0');
   const [inUS, setinUS] = React.useState(false);
+  const [ref, setRef] = React.useState(null);
+
+  // console.log('ref@@@@', ref?.focus());
```
Comment (IAmKio) [inline src/screens/AddCash/AddCash.js:76]: Could we remove any dead code please

File: src/screens/Exchange/ExchangeConfirm/ExchangeConfirm.js [hunk 4]
```diff
@@ -150,7 +150,7 @@ const ExchangeConfirmScreen = () => {
 
           {highFee && (
             <WarningBlock
-              text={t('transactions.highGasFee.warningLabel')}
+              text={t('transactions.highGasFee.ethereumNetworkFees')}
```
Comment (IAmKio) [inline src/screens/Exchange/ExchangeConfirm/ExchangeConfirm.js:153]: This was set as `warningLabel` as the message is blockchain agnostic, whereas `ethereumNetworkFees` will display a message about the Ethereum network when the high gas fee may be on Polygon, for example. See: https://github.com/pillarwallet/pillarwallet/pull/3042

File: src/screens/WalletMigrationArchanova/Review/MigrationSummary.js [hunk 5]
```diff
@@ -124,7 +124,7 @@ function MigrationSummary({ etherspotAccount, archanovaAccount, items, feeInEth,
 
       {feeInEth && highFee && (
         <WarningBlock
-          text={tRoot('transactions.highGasFee.warningLabel')}
+          text={tRoot('transactions.highGasFee.ethereumNetworkFees')}
```
Comment (IAmKio) [inline src/screens/WalletMigrationArchanova/Review/MigrationSummary.js:127]: Same as above: https://github.com/pillarwallet/pillarwallet/pull/3042

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor changes but great work all round! Looking forward to seeing these changes in the app! 👍

File: src/locales/zh-CN/common.json (hunk 2 above)
Comment (divyaBacancy) [reply src/locales/zh-CN/common.json:125]: yes

File: src/components/legacy/Button/Button.js (hunk 1 above)
Comment (divyaBacancy) [reply src/components/legacy/Button/Button.js:225]: I'll remove it

File: src/screens/AddCash/AddCash.js (hunk 3 above)
Comment (divyaBacancy) [reply src/screens/AddCash/AddCash.js:76]: I'll remove it

File: src/screens/WalletMigrationArchanova/Review/MigrationSummary.js (hunk 5 above)
Comment (divyaBacancy) [reply src/screens/WalletMigrationArchanova/Review/MigrationSummary.js:127]: Okay, I'll revert it.

File: src/screens/Exchange/ExchangeConfirm/ExchangeConfirm.js (hunk 4 above)
Comment (divyaBacancy) [reply src/screens/Exchange/ExchangeConfirm/ExchangeConfirm.js:153]: Okay, I'll revert it.

**Review by IAmKio — APPROVED** [review]
LGTM!

**Pull request merged**